symbiotic-build.sh

```diff
@@ -532,7 +534,7 @@ if [ $FROM -le 7 ]; then
 		include/symbiotic.h \
 		lib.c \
 		memalloc.c \
-		LLVM_SLICER_VERSION \
+		#LLVM_SLICER_VERSION \
```

Comment:
This must be moved at the end of list or deleted thoroughly; it can break the script:

```
./symbiotic-build.sh: line 539: LLVM_NEW_SLICER_VERSION: command not found
```